Orange/data/domain.py

```diff
@@ -211,6 +211,9 @@ def variables(self):
     def metas(self):
         return self._metas
 
+    def where(self, condition=Variable):
```

Comment:
Could you do this in a separate PR?
While I agree that this change is neatly hidden here, I would prefer to have modifications to Variables / Domain in a separate, clearly marked PR (where we can discuss different alternatives).

Reply:
👍